sofl/main.py

```
@@ -48,6 +48,7 @@
 from sofl.importer.lutris_source import LutrisSource
 from sofl.importer.retroarch_source import RetroarchSource
 from sofl.importer.steam_source import SteamSource
+
 # Импорт необходим для доступа к классу через globals() в методе get_source_name
 from sofl.importer.onlinefix_source import OnlineFixSource
```

Ruff (0.12.2)

`sofl.importer.onlinefix_source.OnlineFixSource` imported but unused

Remove unused import: `sofl.importer.onlinefix_source.OnlineFixSource`

(F401)